src/olympia/migrations/1004-add-theme-canned-responses.sql

```diff
@@ -0,0 +1,16 @@
+-- sort_group is one of the `actions[]` keys from ReviewHelper
+-- type is CANNED_RESPONSE_THEME = 2
+
```

Comment:
Can you also `DELETE` existing canned responses with that type ? Prod looks ok but there are some left on dev at least.

Reply:
Actually, I didn't realise there were any left anywhere :/
(It was quite possibly me who deleted the webapp canned responses off prod back in the day and forgot about dev/stage).  Should we change type to `4` to avoid the needless potential conflict?

Reply:
I don't particularly care about the conflict as long as we delete the existing responses beforehand (which is a good thing to do anyway)

Reply:
https://github.com/mozilla/addons-server/pull/7805/commits/7665f10e1ceba7b8e469b457db7439305bcc7f2f